Scripts/DBotPredictPhishingWords/dbot_predict_phishing_words_test.py

```diff
@@ -0,0 +1,69 @@
+import demistomock as demisto
+from collections import defaultdict
+from DBotPredictPhishingWords import get_model_data, predict_phishing_words
+
+
+def get_args():
+    args = defaultdict(lambda: "yes")
+    args['encoding'] = 'utf8'
+    args['encoding'] = 'utf8'
+    args['removeNonEnglishWords'] = 'no'
+    return args
+
+
+def executeCommand(command, args=None):
+    if command == 'getList':
+        return [{'Contents': "ModelDataList", 'Type': 'note'}]
+    elif command == 'getMLModel':
+        return [{'Contents': {'modelData': "ModelDataML"}, 'Type': 'note'}]
+    elif command == 'WordTokenizerNLP':
+        if args.get('hashWordWithSeed'):
+            return [{'Contents': {'originalText': 'word1 word2 word3',
+                                  'tokenizedText': "word1 word2 word3", 'hashedTokenizedText': '23423 432432 12321'},
+                     'Type': 'note'}]
+        else:
+            return [{'Contents': {'originalText': 'word1 word2 word3',
+                                  'tokenizedText': "word1 word2 word3"},
+                     'Type': 'note'}]
+    elif command == 'HighlightWords':
+        return [{'Contents': 'word1 word2 word3', 'Type': 'note'}]
+
+
+def test_get_model_data(mocker):
```

Comment:
can we add to the test a case in which the model does not contain any of the input words?

Reply:
added